plan/client.go

```diff
@@ -113,13 +114,24 @@ func (c Client) Update(id string, params *stripe.PlanParams) (*stripe.Plan, erro
 
 // Del removes a plan.
 // For more details see https://stripe.com/docs/api#delete_plan.
-func Del(id string) (*stripe.Plan, error) {
-	return getC().Del(id)
+func Del(id string, params *stripe.PlanParams) (*stripe.Plan, error) {
+	return getC().Del(id, params)
 }
 
-func (c Client) Del(id string) (*stripe.Plan, error) {
+func (c Client) Del(id string, params *stripe.PlanParams) (*stripe.Plan, error) {
+	var body *stripe.RequestValues
+	var commonParams *stripe.Params
+
+	if params != nil {
+		body = &stripe.RequestValues{}
+
+		params.AppendTo(body)
+		commonParams = &params.Params
+	}
+
 	plan := &stripe.Plan{}
-	err := c.B.Call("DELETE", "/plans/"+url.QueryEscape(id), c.Key, nil, nil, plan)
+
+	err := c.B.Call("DELETE", fmt.Sprintf("/plans/%v", id), c.Key, body, commonParams, plan)
```

Comment:
Can we make sure that we keep the `QueryEscape` around here? It looks like it came in for a reason in 98211760beb526dbc58cd91b408488c0ce6929ed.